help/develop/getting-started.md

```diff
@@ -3,208 +3,215 @@
-*Take it fast.* If you are already somewhat knowledgeable with the platform, maybe due to
-previous demos we had for you or having watched some of our
-[youtube](https://www.youtube.com/channel/UCXPHEjOd4gyZ6m6Ji-iOBYg) videos,
-you can take a fast track:   
-> Watch the [20 minutes to an app](https://www.youtube.com/watch?v=tVwpRB8fikQ) video refresher,
-> and proceed directly to the programming [Exercises](exercises.md).  
->  
-When you have time, check
-[Daily Source of Information](#daily-sources-of-information) to know where to post questions
-and get info on the platform updates. Return to this document time to time to study
-platform _in depth_.
-
-*Take it step by step.* In case you prefer a clear start and dive into the platform
-stepwise, we recommend you to follow the below *Table of Contents*:
+If you already know something about Datagrok (probably thanks to our [YouTube channel]), you can take a fast track:
+
+* Have a look at the [20-minute introduction to a Datagrok application] for a refresher, and then proceed to the
+  [JavaScript programming exercises](exercises.md) that explain how to use Datagrok with JavaScript.
+* Check out the [Daily Sources of Information](#daily-sources-of-information), where you can post questions and get
+  information about the platform updates. Return to Daily Sources of Information from time to time to study Datagrok in
+  depth.
+
+In case you want a clean start, review the following sections one by one, in order:
 
 1. [Daily Sources of Information](#daily-sources-of-information)
-2. [Datagrok Video Walkthrough](#datagrok-video-walkthrough)
-3. [Before you start: a Quick F.A.Q.](#faq-frequently-asked-questions)
-4. Datagrok Tutorials: [read here first](#datagrok-tutorials-and-exercises), [start directly](https://public.datagrok.ai/)
-5. Datagrok Exercises [read here first](#datagrok-tutorials-and-exercises), [start directly](exercises.md)
-6. Datagrok Architecture: [read here first](#datagrok-architecture), [start directly](admin/architecture.md)
-7. Extending and customizing Datagrok: [read here first](#extending-and-customizing-datagrok), [start directly](https://datagrok.ai/help/develop/extending-and-customizing)
-8. Building Datagrok Applications: [read here first](#datagrok-application-development), [start directly](how-to/build-an-app.md)
+2. [Datagrok video walkthrough](#datagrok-video-walkthrough)
+3. [Before you start: FAQ section](#frequently-asked-questions)
+4. [Datagrok tutorials](#datagrok-tutorials-and-exercises) or [start using the public version of Datagrok]
+5. [Datagrok exercises](#datagrok-tutorials-and-exercises) or [start directly](exercises.md)
+6. [Datagrok Architecture](#datagrok-architecture) or [start directly](admin/architecture.md)
+7. [Extending and customizing Datagrok](#extending-and-customizing-datagrok) or [start extending and customizing Datagrok]
+8. [Building Datagrok Applications](#datagrok-application-development) or [start directly](how-to/build-an-app.md)
```

Comment:
We are mixing two title styles here: half of the points have all words capitalized, the others don't. Typically, we refer to article names in the documentation as follows: Datagrok Architecture or Building Datagrok Applications. So I suggest not changing this part.

Reply:
Actually, the original documentation isn't consistent. So we just need to decide how to name the sections in all documents: use title-style capitalization or sentence-style capitalization.

Reply:
That's true. I think we try to use title-style capitalization in section headings, but let's just ask the advice of @dskatov and @skalkin and decide on the convention together.